.github/workflows/main.yaml

```diff
@@ -48,6 +48,11 @@ jobs:
           go-version-file: go.mod
           check-latest: true
 
+      - name: Setup Terraform with specified version on the runner
```

Comment:
Not sure why this started breaking, but we get a complaint about missing terraform binary, so I added a step to install it.